src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -551,8 +554,7 @@ private bool SetFrame(float left, float top, float right, float bottom)
                                                          " top:" + _layoutPositionData.Top +
                                                          " right:" + _layoutPositionData.Right +
                                                          " bottom:" + _layoutPositionData.Bottom );
-                Container onwerContainer = Owner.GetParent();
-                View onwerView = onwerContainer is Layer ? new View(Layer.getCPtr(onwerContainer).Handle, false) : onwerContainer as View;
+                View onwerView = Owner.GetParent() as View;
```

Comment:
Layer is not a View.
If Parent is Layer it is not safe to cast directly from Layer to View.

Reply:
As I know, `as` operation is safe enough when casting failed. If it is not safe,.. We have to fix a lot of codes.

Reply:
I think it is safe code. If the parent is Layer, there is no additional work required.